src/Core/Context.cpp

```diff
@@ -102,7 +102,13 @@ void Context::Update() {
     Util::Input::Update();
     SDL_GL_SwapWindow(m_Window);
     glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+
+    if (FPS_CAP != 0) {
+        constexpr double frameTime = 1000 / static_cast<double>(FPS_CAP);
+        SDL_Delay(static_cast<Uint32>(frameTime - Util::Time::GetDeltaTime()));
+    }
```

Comment:
Having `FPS_CAP` as `0` triggers "Constexpr variable 'frameTime' must be initialized by a constant expression division by zero". error. 
```suggestion
constexpr double frameTime =
        FPS_CAP != 0 ? 1000 / static_cast<double>(FPS_CAP) : 0;
    SDL_Delay(static_cast<Uint32>(frameTime - Util::Time::GetDeltaTime()));
```